napari_clusters_plotter/_plotter.py

```diff
@@ -323,6 +425,42 @@ def update_axes_list(self):
         self.plot_y_axis.setCurrentIndex(former_y_axis)
         self.plot_cluster_id.setCurrentIndex(former_cluster_id)
 
+    def make_cluster_overlay_img(
```

Comment:
Could you please make a function from this method as it seems that it is static and could live in the _utilities?